activemq-console/src/main/java/org/apache/activemq/console/command/store/StoreBackup.java

```diff
@@ -178,7 +194,19 @@ public boolean recoverMessage(Message message) throws IOException {
                         return true;
                     }
                 };
-                if(offset != null) {
+                if(startMsgId != null || endMsgId != null) {
+                    System.out.println("Backing up from startMsgId: " + startMsgId + " to endMsgId: " + endMsgId);
```

Comment:
I am thinking for System.out it may be better to use slf4j and log4j2 and configure that for writing to stdout vs directly using System.out.println

Reply:
This was simply following the previous approach used by the StoreExport command. I agree, using logger is better and I'll make this change.

I confirmed other commands are using a logger.